Monika After Story/game/event-handler.rpy

```diff
@@ -332,6 +332,98 @@ init python:
         unlockEvent(eventdb.get(evlabel, None))
 
 
+    def isFuture(ev, date=None):
+        """
+        Checks if the start_date of the given event happens after the
+        given time.
+
+        IN:
+            ev - Event to check the start_time
+            date - a datetime object used to check against
+                If None is passed it will check against current time
+                (Default: None)
+
+        RETURNS:
+            True if the Event's start_date is in the future, False otherwise
+        """
+
+        # sanity check
+        if ev is None:
+            return False
+
+        # if no date is passed
+        if date is None:
+            date = datetime.datetime.now()
```

Comment:
use `datetime.date.today()` since the date properties can only be `date` objects.